### PR TITLE
docs(preset): refactor jetpack `1.0.0`

### PR DESCRIPTION
**Pull request by Indyandie**
#### Description

Update the [jetpack preset](https://starship.rs/presets/jetpack).

#### Motivation and Context

Simplicity...

#### Screenshots (if appropriate):

jetpack

#### How Has This Been Tested?

- [x] I have tested using **MacOS**
- [x] I have tested using **Linux**
- [ ] I have tested using **Windows**

#### Checklist:

- [x] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.

### CODE REVIEW

File: docs/public/presets/toml/jetpack.toml [hunk 1]
```diff
@@ -79,53 +83,61 @@ $openstack\
...
-error_symbol = "[⊘](italic purple)"
-
-[env_var.VIMSHELL] # vim subshell
+success_symbol = "[◎](bold italic bright-yellow)"
+error_symbol = "[○](blink italic purple)"
+# vicmd_symbol = "[■](italic dimmed green)"
+# not supported in zsh
+# vimcmd_replace_one_symbol = "◌"
+# vimcmd_replace_symbol = "□"
+# vimcmd_visual_symbol = "▼"
```
Comment (davidkna) [inline docs/public/presets/toml/jetpack.toml:102]: Why are theses commented?

File: docs/public/presets/toml/jetpack.toml [hunk 2]
```diff
@@ -79,53 +83,61 @@ $openstack\
...
+symbol = ' '
 
 [character]
 format = "$symbol "
-success_symbol = "[◉](bold italic bright-yellow)"
-error_symbol = "[⊘](italic purple)"
-
-[env_var.VIMSHELL] # vim subshell
+success_symbol = "[◎](bold italic bright-yellow)"
+error_symbol = "[○](blink italic purple)"
```
Comment (davidkna) [inline docs/public/presets/toml/jetpack.toml:97]: Please avoid `blink`

File: docs/public/presets/toml/jetpack.toml [hunk 3]
```diff
@@ -1,14 +1,20 @@
-format = """
-$cmd_duration\
+# version: 1.0.0
+
+add_newline = true
+scan_timeout = 500
+command_timeout = 1000
```
Comment (davidkna) [inline docs/public/presets/toml/jetpack.toml:5]: This should be set by users themselves rather than by any preset.

```suggestion
```

**Review by davidkna — COMMENTED** [review]
The changes feel a bit drastic, but this case should be fine since you are the original submitter. Nevertheless, I have made a few suggestions.

File: docs/public/presets/toml/jetpack.toml (hunk 1 above)
Comment (Indyandie) [reply docs/public/presets/toml/jetpack.toml:102]: Taplo was not happy. I'll remove these lines.

File: docs/public/presets/toml/jetpack.toml (hunk 3 above)
Comment (Indyandie) [reply docs/public/presets/toml/jetpack.toml:5]: agreed

File: docs/public/presets/toml/jetpack.toml (hunk 1 above)
Comment (davidkna) [reply docs/public/presets/toml/jetpack.toml:100]: I think it needs to be `vimcmd_symbol` (it was renamed and taplo doesn't know about the `vicmd_symbol` alias).

Comment (Indyandie) [reply docs/public/presets/toml/jetpack.toml]: Oh wow! I couldn't see the difference to save my life. That `m` was invisible to me. Thanks for pointing it out! :bow:

**Comment by Indyandie** [comment]
@davidkna I just realized I didn't update the preset image. Doing that now.

**Review by davidkna — APPROVED** [review]

**Comment by andytom** [comment]
Thank you for your contribution @Indyandie and thanks for reviewing @davidkna.

**Pull request merged**